screengrab/screengrab-lib/build.gradle

```diff
@@ -51,4 +51,5 @@ dependencies {
     compile 'com.android.support.test:rules:0.5'
     compile 'com.android.support.test.espresso:espresso-core:2.2.2'
     compile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.2'
+    compile 'com.jraska:falcon:1.0.4'
```

Comment:
Hey @Macarse, is there maybe a way to only require this if the Falcon thing is actually used, so do it on runtime? Would This is the only change that will affect users not using this strategy, and would make merging much easier of course.

Reply:
@janpio I think fastlane should include falcon as a dependency. If Falcon is not added, using the uiautomator implementation will have race condition bugs.

Reply:
Honestly, I would drop the other 2 implementations and force everyone to use Falcon.